.github/workflows/test-changed-firestore.yml

```diff
@@ -28,27 +34,115 @@ jobs:
           cp config/ci.config.json config/project.json
           yarn
       - name: build
-        run: yarn build:changed firestore
-      - name: Run tests if firestore or its dependencies has changed
-        run: yarn test:changed firestore
+        id: build
+        run: yarn build:changed firestore | egrep "Skipping all"
+        # Continue when "Skipping all" is not found
+        continue-on-error: true
+      - name: set output
+        # This means "Skipping all" was not found
+        if: steps.build.outcome != 'success'
+        id: set-output
+        run: echo "CHANGED=true" >> "$GITHUB_OUTPUT";
+      - name: Archive build
+        if: ${{ !cancelled() && steps.build.outcome != 'success' }}
+        run: |
+          tar -cf build.tar --exclude="\.git" .
+          gzip build.tar
+      - name: Upload build archive
+        if: ${{ !cancelled() && steps.build.outcome != 'success' }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: build.tar.gz
+          path: build.tar.gz
+          retention-days: ${{ env.artifactRetentionDays }}
+
+  compat-test-chrome:
+    name: Test Firestore Compatible
+
+    runs-on: ubuntu-latest
+
+    needs: build
+    if: ${{ needs.build.outputs.changed == 'true'}}
```

Comment:
The output is stored as `CHANGED` above. Is case not important?

Reply:
the output is actually `changed`, as defined in line 14:

`      changed: ${{ steps.set-output.outputs.CHANGED }}`.

Reply:
got it, thanks!